packages/suite/src/hooks/suite/useDaysTo.ts

```diff
@@ -4,20 +4,28 @@ import {
     selectAccountStakeTransactions,
     selectAccountUnstakeTransactions,
 } from '@suite-common/wallet-core';
-import { ValidatorsQueueState } from 'src/types/wallet/stake';
+import { ValidatorsQueue } from '@suite-common/wallet-core/src/stake/stakeTypes';
 
 const secondsToDays = (seconds: number) => Math.round(seconds / 60 / 60 / 24);
 
 interface UseDaysToParams {
-    validatorsQueue: ValidatorsQueueState;
+    validatorsQueue?: ValidatorsQueue;
     selectedAccountKey: string;
 }
 
 export const useDaysTo = ({ selectedAccountKey, validatorsQueue }: UseDaysToParams) => {
```

Comment:
optional: split to 3 non-hook pure functions

Reply:
82dadfc79a291941d75676190c23fbbf2dbe84bd